egs/tedlium/local/map_acronyms_transcripts.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# convert acronyms in swbd transcript to fisher convention
```

Comment:
You can remove the file map_acronyms_transcripts.py, I think the cantab data is already normalized.

And please let us know the timings by writing them somewhere in the run.sh in a comment.
I assume it didn't take you 11 hours like it did vince62s?